tests/Fixtures/e2e/Test_Frameworks/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
```

Comment:
we're using 4 spaces in the .json files. Could we please use the same style everywhere?

Reply:
@theofidry left same comments in https://github.com/infection/infection/pull/151

leave it with me, I will fix it there

Reply:
@borNfreee Do you mean you will fix it in this PR, or do you want me to fix it?

Reply:
I will fix it in my PR, let's get it merged ;)